inc/saiobject.h

```diff
@@ -58,37 +58,43 @@ typedef struct _sai_object_key_t
 /**
```

Comment:
yes, your understanding is correct. This file is defined to get the objects from SAI. switch id for other than create derived internally in SAI, in this bluk get cases,  caller can not know switch id by looking object id,
since this is implementation specific. 
bulk api's get(defined for warmboot),  it is good to know from which NPU they are getting bulk objects.